sdk/data/azcosmos/cosmos_client.go

```diff
@@ -124,6 +126,7 @@ func newPipeline(authPolicy policy.Policy, gem *globalEndpointManager, options *
 			},
 			PerRetry: []policy.Policy{
 				authPolicy,
+				&clientRetryPolicy{gem: gem},
```

Comment:
Should the flag be wired here?

Reply:
in other SDKs we generally pass the option forward to the GEM, but we technically only use it within the policy so we could

Reply:
I see, because we use the flag within the GEM/LocationCache